akka-distributed-cluster-docs/src/main/paradox/guide/2-service-to-service.md

```diff
@@ -0,0 +1,58 @@
+# Part 2: Service to Service eventing
+
+To implement Service to Service eventing, we will use two services, the shopping cart defined in the previous step and 
+a downstream analytics service. 
+
+Each of the services have their own lifecycle and are deployed separately, possibly in different data centers or
+cloud regions.
+
+## gRPC transport for consuming events
+
+Akka Projection allows for creating read side views, or projections, that are eventually consistent representations
+of the events for an entity. Such views have historically been possible to define in the service that owns the entity,
+for an example of this, you can look at the [popularity projection in the Akka Microservice Guide](https://developer.lightbend.com/docs/akka-guide/microservices-tutorial/projection-query.html).
+
+Service to Service defines a gRPC service in the service where the entity lives and that makes the events available for
+other services to consume with an effectively once delivery guarantee without requiring a message broker in between services.
```

Comment:
Here is that description that I asked about above, maybe it's just the heading `## gRPC transport for consuming events`. Can that heading be removed?

Reply:
Now pulled up top